client/src/components/leftSidebar/index.js

```diff
@@ -8,47 +8,45 @@ import DynamicScatterplot from "../scatterplot/scatterplot";
 import TopLeftLogoAndTitle from "./topLeftLogoAndTitle";
 import AuthN from "./AuthN";
 
-@connect((state) => ({
```

Comment:
Since we're not using JavaScript Class anymore, we can't use the decorator as well, so refactored this to use the plain old `connect()` below:

```tsx
const mapStateToProps = (state) => ({
  scatterplotXXaccessor: state.controls.scatterplotXXaccessor,
  scatterplotYYaccessor: state.controls.scatterplotYYaccessor,
});

export default connect(mapStateToProps)(LeftSideBar);
```